web/package.json

```diff
@@ -27,13 +27,18 @@
     "@typescript-eslint/parser": "^5.49.0",
     "eslint": "^8.32.0",
     "gts": "^3.1.1",
+    "karma": "^6.4.1",
+    "karma-chrome-launcher": "^3.1.1",
+    "karma-jasmine": "^5.1.0",
+    "karma-webpack": "^5.0.0",
     "ts-loader": "^9.4.2",
     "typescript": "^4.9.5",
     "webpack": "^5.61.0",
     "webpack-cli": "^5.0.1"
   },
   "scripts": {
+    "test": "karma start",
     "pack": "webpack --mode production",
-    "build": "rm -rf dist && tsc --build && cp scalar_pb.js dist && cp scalar_grpc_web_pb.js dist"
+    "build": "rm -rf build && tsc --build && cp scalar_pb.js build && cp scalar_grpc_web_pb.js build"
```

Comment:
the `build` (TypeScript -> JavaScript) output goes to the `build` folder. The dist folder is now for one file bundle.